src/endpoints/criarEstudante.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express";
+import connection from "../connection";
+import { estudantes } from "../types";
+
+export const criarEstudante = async (req: Request, res: Response) => {
+    let errorCode: number = 400;
+
+    try {
+        const input: estudantes ={
+            id: req.body.id,
+            nome: req.body.nome,
+            email: req.body.email,
+            data_nascimento: req.body.data_nascimento,
+            hobbies: req.body.hobbies,
+            turma_id: req.body.turma_id
+        }
+
+        if (!input.id || !input.nome || !input.email || !input.data_nascimento || input.hobbies.length>1) {
```

Comment:
Atenção ao length de hobbies aqui. Com a validação como está, o usuário só pode ter um hobbie por vez, pois se for maior que 1, o erro é atirado